src/controllers/controller_post.rs

```diff
@@ -1,15 +1,13 @@
-use crate::model::models::{get_details_of_post, Post, PostTemplate};
+use crate::model::models::{get_details_of_post, PostTemplate};
 use askama::Template;
 use axum::{
-    extract::{Path, State},
+    extract::Path,
     http::StatusCode,
     response::{Html, IntoResponse},
 };
-use std::sync::Arc;
 
-pub async fn show_post(
-    Path(post_id): Path<String>
-) -> impl IntoResponse {
+pub async fn show_post(Path(post_id): Path<String>) -> impl IntoResponse {
+    let _map_post: Vec<String> = Vec::new();
```

Comment:
Use `vec![]` instead
```suggestion
    let _map_post: Vec<String> = vec![];
```

Reply:
Changed the definition in the commit "Resolved"